gradio/blocks.py

```diff
@@ -768,28 +769,32 @@ def clear(self):
         self.children = []
         return self
 
-    def configure_queue(
+    @document()
+    def queue(
         self,
-        live_queue_updates: bool = True,
         concurrency_count: int = 1,
-        data_gathering_start: int = 30,
-        update_intervals: int = 5,
-        duration_history_size: int = 100,
+        status_update_rate: float | str = "auto",
```

Comment:
Better to use `float | Literal["auto"]` for the type hint

Reply:
not supported in 3.7